man/memtier.7

```diff
@@ -11,11 +11,38 @@ command {arguments ...}
 .B TODO
 
 .SH "ENVIRONMENT"
-
-The behavior of Memtier library is controlled by the following environment
-variables.
-
-TODO
+.TP
+.B MEMKIND_MEM_TIERING_CONFIG
+Sets the memkind tiering functionality. Passed as a comma separated list of
+.I memkind_tier
+configuration strings. The configuration has to be passed in the following format:
+"kind_name:path_to_pmem:pmem_file_size:ratio"
+, where:
+.IP
+.BR
+kind_name - the name of a kind used in tiering. Allowed kind names are 'DRAM' and 'FS_DAX'.
+.IP
+.BR
+path_to_pmem - the path to the location where pmem file will be created. The path has to exist.
+.IP
+.BR
+pmem_file_size - the size of pmem file, it's the maximum size of total allocations to PMEM.
+The accepted formats are: 1, 1KB, 1MB, 1GB. Valid values of size are from zero to the maximum value of
+.I size_t
+.IP
+.BR 
+ratio - the part of ratio tied to the given kind. It's represented as an
```

Comment:
This is a bit confusing. Are they supposed to be a nominator and denominator of a fraction? Why not use a simple percentage ratio? Or maybe only indicate how many allocations should got to one of the kinds and all the rest to the other one?